cristian-bars/precurse/Calculadora/calculator-pro.js

```diff
@@ -0,0 +1,79 @@
+function operaciones(opera){
+    let numberList = [];
+    let newNumber;
+    do {
+        newNumber = prompt('Enter a number or press cancel to stop');
+        if (newNumber == ""){
+            alert("Dejaste en blanco el primer numero");
+        }else if (isNaN(newNumber)){
+            alert("Entraste letras en vez de numeros");
+            newNumber = "";
+        }else if (newNumber !== null){
+            console.log(newNumber);
+            numberList.push(Number(newNumber))
+        }
+        
+        console.log(numberList);
+    } while(newNumber !== null)
+    
```

Comment:
Apply delete empty line break in all lines except:
1. Before a function declaration.
2. Between Function declaration section && Variables or Function summon section.